### DISCUSSION TITLE
Are profile-specific actions supported (or any kind of conditional enabling/disabling)?

### DISCUSSION THREAD

**Discussion by ev-dev** [microsoft/terminal#15531] [Q&A] [answered]
Hi does anyone know if it is currently possible (preview version) to configure an action in the settings json which would only be enabled within a specific profile tab/pane?

For instance, when I have PowerShell open, I prefer to have Ctrl+V unbound so that multi-line paste is handled by pwsh. But simply unbinding Ctrl+V means I cannot use that combo to paste in other profiles that aren't pwsh based (like WSL/bash ones). 

I was hoping actions had some kind of property for specifying profiles, but I couldn't seem find it in the settings schema nor could I find info about such a thing in any issues/commments here, anyone know more?

**Comment by zadjii-msft** [comment, marked as answer]
Not currently, but that's tracked in #5790

**Reply by ev-dev** [reply]
Ah nice thank you I should've been able to find that on my own! 

You're always so helpful (I notice you all over MS's OSS repos), really appreciate all your involvement in the community 🙌
